clang/lib/Sema/SemaDecl.cpp

```diff
@@ -12806,6 +12810,19 @@ namespace {
         HandleValue(Arg->IgnoreParenImpCasts());
     }
 
+    void VisitLambdaExpr(LambdaExpr *E) {
+      if (!isInCXXOperatorCall) {
+        Inherited::VisitLambdaExpr(E);
+        return;
+      }
+
+      for (const auto &init : E->capture_inits())
```

Comment:
```suggestion
      for (const Expr *Init : E->capture_inits())
```
Note the not using auto per coding standard, and the capital 'I'.

Reply:
Fine, I'll pay attention to that in the future.